hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -260,6 +262,18 @@ def on_edn_urnd_step(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     return None
 
 
+def on_set_keymgr_value(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+    if len(args) != 3:
+        raise ValueError('set_keymgr_value expects exactly 1 argument. Got {}.'
+                         .format(args))
+    key0 = read_word('key0', args[0], 384)
+    key1 = read_word('key1', args[1], 384)
+    valid = bool(args[2])
+    sim.state.set_keymgr_value(key0, key1, valid)
```

Comment:
```suggestion
    valid = read_word('valid', args[2], 1) == 1
    sim.state.set_keymgr_value(key0, key1, valid)
```

Reply:
This still needs changing (otherwise we won't get an error if we accidentally send 0x1234 for the `valid` argument).